Sources/NIOSSL/TLSConfiguration.swift

```diff
@@ -393,3 +393,27 @@ public struct TLSConfiguration {
                                 renegotiationSupport: renegotiationSupport)
     }
 }
+
+// MARK: Hashable
+extension TLSConfiguration: Hashable {
+    public static func == (lhs: TLSConfiguration, rhs: TLSConfiguration) -> Bool {
+        lhs.hashValue == rhs.hashValue
```

Comment:
That one’s not quite correct. Hash values of unequal things may be the same. You’ll have to compare all the individual fields.

Reply:
Ah okay, that makes sense yea!